crates/subspace-networking/src/create.rs

```diff
@@ -73,55 +62,23 @@ impl Default for RelayLimitSettings {
             max_circuits_per_peer: default_relay_config.max_circuits_per_peer,
             max_circuit_duration: default_relay_config.max_circuit_duration,
             max_circuit_bytes: default_relay_config.max_circuit_bytes,
-
-            // Copied from the  RelayConfig::default() implementation:
-            // For each peer ID one reservation every 2 minutes with up to 30 reservations per hour.
-            reservation_rate_limit_per_peer: rate_limiter::GenericRateLimiterConfig {
-                limit: NonZeroU32::new(30).expect("30 > 0"),
-                interval: Duration::from_secs(60 * 2),
-            },
-            // For each IP address one reservation every minute with up to 60 reservations per hour.
-            reservation_rate_limit_per_ip: rate_limiter::GenericRateLimiterConfig {
-                limit: NonZeroU32::new(60).expect("60 > 0"),
-                interval: Duration::from_secs(60),
-            },
-            // For each source peer ID one circuit every 2 minute with up to 30 circuits per hour.
-            circuit_src_rate_limit_per_peer: rate_limiter::GenericRateLimiterConfig {
-                limit: NonZeroU32::new(30).expect("30 > 0"),
-                interval: Duration::from_secs(60 * 2),
-            },
-            // For each source IP address one circuit every minute with up to 60 circuits per hour.
-            circuit_src_rate_limit_per_ip: rate_limiter::GenericRateLimiterConfig {
-                limit: NonZeroU32::new(60).expect("60 > 0"),
-                interval: Duration::from_secs(60),
-            },
         }
     }
 }
 
 impl RelayLimitSettings {
     /// Converts `RelayLimitSettings` to libp2p `relay::Config`.
     pub fn to_relay_config(self) -> RelayConfig {
-        let reservation_rate_limiters = vec![
```

Comment:
We removed rate limiters, but I don't think we have any restriction yet on who can use this circuit relay. The intention is to limit it just to the local peers within the same process. Probably worth having a TODO so we don't forget about it.